ledger-service/http-json/src/test/scala/com/digitalasset/http/CliSpec.scala

```diff
@@ -27,6 +27,50 @@ final class CliSpec extends AnyFreeSpec with Matchers {
   val sharedOptions =
     Seq("--ledger-host", "localhost", "--ledger-port", "6865", "--http-port", "7500")
 
+  "LogLevel" - {
+    import ch.qos.logback.classic.{Level => LogLevel}
+
+    "should get the error log level from the command line argument when provided" in {
+      val config = configParser(Seq("--log-level", "error") ++ sharedOptions)
```

Comment:
You could probably factor out a helper function of type `(String, LogLevel) => Assertion` here. The code is almost identical between the different tests.

Reply:
adressed, hope this is fine